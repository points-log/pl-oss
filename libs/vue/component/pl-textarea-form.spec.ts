import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLTextareaForm from './pl-textarea-form.vue';

Vue.use(Vuetify);

describe('PLTextareaForm', () => {
  it('should match the snapshot', () => {
    const stubs = {
      'pl-cancel-btn': true,
      'pl-done-btn': true,
      'pl-textarea': true,
    };
    const propsData = { title: 'Title' };
    const wrapper = shallowMount(PLTextareaForm, { propsData, stubs });
    expect(wrapper.element).toMatchSnapshot();
  });
});
