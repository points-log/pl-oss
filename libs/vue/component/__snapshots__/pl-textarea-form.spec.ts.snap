// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLTextareaForm should match the snapshot 1`] = `
<v-form-stub>
  <v-card-stub
    loaderheight="4"
    tag="div"
  >
    <v-card-title-stub>
      Title
    </v-card-title-stub>
    <v-card-text-stub>
      <pl-textarea-stub
        autofocus="autofocus"
      />
    </v-card-text-stub>
    <v-card-actions-stub>
      <v-spacer-stub />
      <pl-cancel-btn-stub />
      <pl-done-btn-stub />
    </v-card-actions-stub>
  </v-card-stub>
</v-form-stub>
`;
